cupyx/_ufunc_config.py

```diff
@@ -88,6 +119,23 @@ def seterr(*, divide=None, over=None, under=None, invalid=None, linalg=None):
     return old_state
 
 
+def seterrcall(fallback_mode=None):
+    """
+    TODO: Write docs.
+    """
+    old_state = geterrcall()
+
+    if fallback_mode is not None:
+        if not callable(fallback_mode):
+            if not hasattr(fallback_mode, 'write') or \
+               not callable(fallback_mode.write):
+                raise ValueError('Only callable can be used as Callback')
+
+    _config.fallback_mode_callback = fallback_mode
```

Comment:
This function is not only for fallback-mode.
```suggestion
    _config.seterrcall_func = fallback_mode
```